src/include/kernel/tokens.h

```diff
@@ -106,4 +106,7 @@
 /* ESAI */
 #define SOF_TKN_IMX_ESAI_MCLK_ID		1100
 
+/* SAI */
+#define SOF_TKN_IMX_SAI_MCLK_ID			1400
+
```

Comment:
After my comment on Linux PR this should be modified and kept in sync.

Reply:
done